.github/workflows/releaseArtifacts.yml

```diff
@@ -14,6 +14,8 @@ jobs:
     permissions:
       contents: write
       packages: write
```

Comment:
I don't see any content or package updates. Is `write` access needed? Wouldn't `read` be sufficient?